src/apps/tron/get_address.py

```diff
@@ -0,0 +1,24 @@
+from trezor.crypto.curve import secp256k1
+from trezor.messages.TronAddress import TronAddress
+
+from .helpers import get_address_from_public_key
+
+from apps.common import seed
+from apps.common.layout import show_address, show_qr
+
+
+async def get_address(ctx, msg):
+    address_n = msg.address_n or ()
```

Comment:
in all cases please use address validation, see e.g. Ethereum:
https://github.com/trezor/trezor-core/blob/master/src/apps/ethereum/get_address.py#L12
https://github.com/trezor/trezor-core/blob/master/src/apps/ethereum/address.py#L38